### PR TITLE
U/fjammes/dm 3951

### PR DESCRIPTION
**Pull request by fjammes**

### CODE REVIEW

File: core/modules/qana/QservRestrictorPlugin.cc [hunk 1]
```diff
@@ -601,37 +580,29 @@ QservRestrictorPlugin::applyLogical(query::SelectStmt& stmt,
     // Prepare to patch the WHERE clause
     query::WhereClause& wc = stmt.getWhereClause();
 
-    std::shared_ptr<query::QsRestrictor::PtrVector const> restrsPtr = wc.getRestrs();
     query::AndTerm::Ptr originalAnd(wc.getRootAndTerm());
-    query::QueryContext::RestrList restrictors;
-    // Now handle the explicit restrictors
-    if (restrsPtr && not restrsPtr->empty()) {
+    query::QueryContext::RestrList ctx_restrictors;
```
Comment (jbecla) [inline core/modules/qana/QservRestrictorPlugin.cc]: i think we are told to use camelcase for naming variables (this applies to wc_restrictors below too)

File: core/modules/qana/QservRestrictorPlugin.cc [hunk 2]
```diff
@@ -601,37 +580,29 @@ QservRestrictorPlugin::applyLogical(query::SelectStmt& stmt,
...
         // At least one table should exist in the restrictor entries
         if (entries.empty()) {
             throw AnalysisError("Spatial restrictor w/o partitioned table");
         }
 
+        query::AndTerm::Ptr newTerm = std::make_shared<query::AndTerm>();
+        // spatial restrictions
         // Now, for each of the qserv restrictors:
-        for (auto const i : restrs) {
+        for (auto const restrictor_i : *wc_restrictors) {
```
Comment (jbecla) [inline core/modules/qana/QservRestrictorPlugin.cc]: how about restrItr?

File: core/modules/qproc/testQueryAnaIn.cc [hunk 3]
```diff
@@ -96,4 +97,21 @@ BOOST_AUTO_TEST_CASE(CountIn) {
...
+    BOOST_CHECK_EQUAL(context->dominantDb, std::string("LSST"));
+    BOOST_REQUIRE(context->restrictors);
+    BOOST_CHECK_EQUAL(context->restrictors->size(), 1U);
+    BOOST_REQUIRE(context->restrictors->front());
+    QsRestrictor& r = *context->restrictors->front();
+    BOOST_CHECK_EQUAL(r._name, "sIndex");
+    char const* params[] = {"LSST","Object", "objectIdObjTest", "2","3145","9999"};
+    BOOST_CHECK_EQUAL_COLLECTIONS(r._params.begin(), r._params.end(),
+                                  params, params+6);
+
```
Comment (jbecla) [inline core/modules/qproc/testQueryAnaIn.cc]: extra line, remove?

File: core/modules/qana/QservRestrictorPlugin.cc (hunk 1 above)
Comment (fjammes) [inline core/modules/qana/QservRestrictorPlugin.cc]: Ok I switch case, but FYI I can't find this requirement in LSST C++ coding standards. Furthermore, using non-camelcase for naming local variables could increase readability, isn't it?

Comment (ktlim) [inline core/modules/qana/QservRestrictorPlugin.cc]: Standard 3-2.

**Comment by fjammes** [comment]
ok everything done :8ball:

File: core/modules/qana/QservRestrictorPlugin.cc (hunk 1 above)
Comment (fjammes) [inline core/modules/qana/QservRestrictorPlugin.cc]: ok thanks :-)

**Pull request closed without merging**